softlayer/vm/softlayer_vm.go

```diff
@@ -24,7 +24,7 @@ const (
 	softLayerVMtag                 = "SoftLayerVM"
 	ROOT_USER_NAME                 = "root"
 	deleteVMLogTag                 = "DeleteVM"
-	TIMEOUT_TRANSACTIONS_DELETE_VM = 10 * time.Minute
+	TIMEOUT_TRANSACTIONS_DELETE_VM = 60 * time.Minute
```

Comment:
Can you check with @posijon if this increase is needed because you are running it from Beijing? In other words, whether here in U.S. It will be faster? I ask because @posijon's `has_vm` Integration test is about the same but does not take this long...